src/nunavut/cli.py

```diff
@@ -365,6 +367,16 @@ def extension_type(raw_arg: str) -> str:
 
     ''').lstrip())
 
+    parser.add_argument('-upid', '--allow-unregulated-fixed-port-id',
```

Comment:
```suggestion
    parser.add_argument('--allow-unregulated-fixed-port-id',
```

dangerous things should be hard to reach

Reply:
Accepted but I thought you'd appreciate the pun (i.e. the missing "st")

Reply:
This pun requires a user manual. Or maybe it's just me -- I grew up in a town named Stupino which might have made me insensitive to this specific kind of jokes, not sure.